sites/cheerpx/src/content/docs/11-guides/File-System-support.md

```diff
@@ -69,6 +69,65 @@ Files in the WebDevice are accessed relative to the current page's URL. For exam
 > [!note] Note
 > It's important to note that this behavior depends on the current page's URL, as it uses a relative path. For more predictable results, it's recommended to use absolute paths when possible.
 
+### Using `application/octet-stream` for WebDevice directories
+
+To handle files from WebDevices properly, set default_type `application/octet-stream`. This ensures that files are treated as binary data. **Only apply this to WebDevice directories**, or it may cause issues with image loading and JavaScript execution.
```

Comment:
```suggestion
To handle files from WebDevices properly, make sure the `application/octet-stream` MIME type is applied. This ensures that files are treated as binary data. **Only apply this to WebDevice directories**, or it may cause issues with image loading and JavaScript execution.
```

The purpose of this change is to tell the user _what_ is going on, before telling him _how_